### PR TITLE
bugfix for audio pops at startup

### PR DESCRIPTION
**Pull request by kencooke**
SOXR does not return samples while it is buffering at startup.  Now pads with silence instead of playing garbage samples.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
An admin must verify this patch for the pull request build to fire. Type "ok to test" to begin testing this pull request or "add to whitelist" to test this and all future pull requests from this collaborator.

**Comment by ZappoMan** [comment]
add to whitelist

File: libraries/audio-client/src/AudioClient.cpp [hunk 1]
```diff
@@ -418,9 +418,17 @@ soxr_error_t possibleResampling(soxr_t resampler,
                                         numSourceSamples,
                                         sourceAudioFormat, destinationAudioFormat);
 
+                unsigned int odone = 0;
```
Comment (ZappoMan) [inline libraries/audio-client/src/AudioClient.cpp]: coding standard: variable names should not use hungarian notation

Comment (kencooke) [inline libraries/audio-client/src/AudioClient.cpp]: thanks, I'll fix it with another commit

**Comment by hifi-gustavo** [comment]
Build console output for commit: 984ea6f2e27a792a6301334afbfb57774c703a1f available at: https://gist.github.com/998c14e1836f7ebe5142<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: bc5532bd0d224b5261ed36de01a7e2442f0246c3 available at: https://gist.github.com/998c14e1836f7ebe5142<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b245e8245ae63b15acf9ab5be10f0c6d77d0a7eb available at: https://gist.github.com/998c14e1836f7ebe5142<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 505e11f60adcc0daf3587a3edf8f5d5bf4b16560 available at: https://gist.github.com/998c14e1836f7ebe5142<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5793.dmg

**Comment by PhilipRosedale** [comment]
Testing and verified working on Windows.  Shrill noise on connect gone.

**Pull request merged**